tests/network/probes.go

```diff
@@ -83,7 +83,6 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 			Eventually(matcher.ThisVMI(vmi), 2*time.Minute, 2*time.Second).Should(matcher.HaveConditionTrue(v1.VirtualMachineInstanceReady))
```

Comment:
Commit message:
> Currently TCP tests are performed twice.

I find this sentence a bit misleading. It makes the reader understand that TCP probes are tested twice and one of the tests is redundant.
Please consider adjusting the commit message.

nit: typo:
> Remjove HTTP tests and keep TCP tests.